src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests2.cs

```diff
@@ -2120,7 +2120,13 @@ public class C {
             compilation.VerifyDiagnostics(
                 // (4,21): error CS8407: A single-element deconstruct pattern requires a type before the open parenthesis.
                 //         _ = this is (a: 1);
-                Diagnostic(ErrorCode.ERR_SingleElementPositionalPatternRequiresType, "(a: 1)").WithLocation(4, 21)
+                Diagnostic(ErrorCode.ERR_SingleElementPositionalPatternRequiresType, "(a: 1)").WithLocation(4, 21),
+                // (4,21): error CS1061: 'C' does not contain a definition for 'Deconstruct' and no extension method 'Deconstruct' accepting a first argument of type 'C' could be found (are you missing a using directive or an assembly reference?)
+                //         _ = this is (a: 1);
+                Diagnostic(ErrorCode.ERR_NoSuchMemberOrExtension, "(a: 1)").WithArguments("C", "Deconstruct").WithLocation(4, 21),
+                // (4,21): error CS8129: No suitable Deconstruct instance or extension method was found for type 'C', with 1 out parameters and a void return type.
+                //         _ = this is (a: 1);
+                Diagnostic(ErrorCode.ERR_MissingDeconstruct, "(a: 1)").WithArguments("C", "1").WithLocation(4, 21)
```

Comment:
What caused this change (more diagnostics)?

Reply:
There is an error on the enclosing node, but there isn't an error on the deconstruction subpattern. They were together before.